marketplace_standard_app_api/routers/transformation.py

```diff
@@ -127,3 +127,62 @@ async def list_transformation(
 ) -> TransformationListResponse:
     """Retrieve a list of transformations."""
     raise HTTPException(status_code=501, detail="Not implemented.")
+
+
+@router.get(
+    "/{transformation_id}/log",
+    operation_id="getTransformationLog",
+    summary="Get logs for a transformation",
+    responses={
+        404: {"description": "Not found."},
+    },
+)
+async def get_transformation_log(
+    transformation_id: TransformationId,
+) -> Response:
+    """Retrieve an existing transformation log."""
+    raise HTTPException(status_code=501, detail="Not implemented.")
+
+
+@router.get(
+    "/models/get_schema/{modelname}",
+    operation_id="getSchema",
+    summary="Retreive schema of a model registered in the app.",
+    responses={
+        404: {"description": "Not found."},
+    },
+)
+async def get_schema(
+    modelname: ModelName,
+) -> JSONResponse:
+    """Retreive schema of a model registered in the app."""
+    raise HTTPException(status_code=501, detail="Not implemented.")
+
+
+@router.get(
+    "/models/get_example/{modelname}",
+    operation_id="getExample",
+    summary="Retrieve an example for a model registered in the app.",
+    responses={
+        404: {"description": "Not found."},
+    },
+)
+async def get_example(
+    modelname: ModelName,
+) -> JSONResponse:
+    """Retrieve an example for a model registered in the app."""
+    raise HTTPException(status_code=501, detail="Not implemented.")
```

Comment:
I am not too sure about this capability, but:
- The operation (get) should not be encoded in the path.
- For me `/models/{modelname}/example` seems more logical.
- ´getExample´ is a very generic capability name for the very specific task you have in mind

Reply:
This endpoint is specific to reaxpro-workflow usecase and hence I used the same name provided by Matthias. I will change the definition as you suggested.